docs/development-setup.md

```diff
@@ -43,15 +43,17 @@ Make sure Redis and MariaDB server are up and running.
 
    # Database configuration
    DB_TYPE=mysql
-   DB_HOST=localhost
+   DB_HOST=localhost # use value as mariadb in docker
```

Comment:
It should be the container name
`osmo-notify-mariadb`

Reply:
No its not container name, when I tested it it was the service name present in `docker-compose.yml` file which were `mariadb` and `redis` I will instead update these service name.